DataProcessing/datax-host/src/main/scala/datax/host/SparkJarLoader.scala

```diff
@@ -57,7 +59,9 @@ object SparkJarLoader {
 
   def addJar(spark: SparkSession, jarPath: String) = {
     addJarOnDriver(spark, jarPath)
-    spark.sparkContext.addJar(jarPath)
+	val logger = LogManager.getLogger("AddJar to executer")
+    logger.info("fileUrl is " + fileUrl)
+    spark.sparkContext.addJar(fileUrl)
```

Comment:
Does this still work for Spark?

Reply:
Adding a comment?

Reply:
verified that it works with HDInsight and databricks. Also added a comment